webpack.config.js

```diff
@@ -179,18 +191,88 @@ var pluginsProd = [
         NormalModuleReplacementPluginOptionsSVG
     ),
 
-    new webpack.NormalModuleReplacementPlugin(
-        /^css!/,
-        NormalModuleReplacementPluginOptionsCSS
-    ),
+    new ExtractCssChunks({
+        filename: prodMode ? "css/[name].[contenthash].css" : "css/[name].css",
+        chunkFilename: "css/[id].css",
+        moduleFilename: ({ name }) => `${name.replace("js/", "js/css/")}.css`
+        // publicPath: "js"
+    }),
 
+    new HtmlWebpackPlugin({
+        inject: false, // Tags are injected manually in the content below
+        minify: false, // Adjust t/16-schema-upgrade-html.t if prodMode is used,
+        filename: "ui-header.html",
+        excludeChunks: [...Object.keys(lsmbCSS)],
+        templateContent: ({ htmlWebpackPlugin }) =>
+            `<!-- prettier-disable -->\n` +
+            `[%#\n` +
```

Comment:
With this template here, there's little point to have it in `UI/lib`; maybe we can read it from `UI/lib` instead?

Reply:
Nope, we'll need a Perl TT enabled loader for that

Reply:
it took a bit before I understood that remark.

different question: maybe we need to remove it from `UI/lib/` so there's no confusion about which content is the actual content?

Reply:
If it is limited to setup, login and logout, we don't need it anymore

Reply:
> it took a bit before I understood that remark.
> 
> different question: maybe we need to remove it from `UI/lib/` so there's no confusion about which content is the actual content?

We still inject values from Perl TT and we need Webpack to adjust the included CSS & JS file names, which are hashed for long-term caching. I cannot load a file directly because validate the syntax and abort on TT statements. But I can inject code directly and that the solution I picked for the moment.

Reply:
> If it is limited to setup, login and logout, we don't need it anymore

I think it applies to all pages in `UI/setup/`, because we are not using the SPA-approach for setup.pl yet.

Reply:
Yes it does.